theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-contribution.ts

```diff
@@ -123,13 +127,17 @@ export class TraceViewerContribution extends WidgetOpenHandler<TraceViewerWidget
         registry.registerCommand(StartServerCommand, {
             execute: async () => {
                 try {
-                    await this.traceServerConfigService.startTraceServer(this.path, this.port);
-                    this.messageService.info(`Trace server started successfully on port: ${this.port}.`);
+                    await this.traceServerConfigService.startTraceServer(this.path, this.cliArgs);
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    const port = this.cliArgs!.match(/(?<=port=)\d+/g)![0];
+                    this.messageService.info(`Trace server started successfully on port: ${port}.`);
```

Comment:
The message should not mention the port. Either just mention `Trace Server started successfully` or add the whole command-line executed: `Trace Server started successfully: ./tracecompass-server -vmargs -Dtraceserver.port=8080`.

Make it consistent with the above with toast message in the other file (TraceExplorerPlaceholderWidget) above.